contracts/FlatLaunchpeg.sol

```diff
@@ -196,8 +206,13 @@ contract FlatLaunchpeg is BaseLaunchpeg, IFlatLaunchpeg {
             block.timestamp < publicSaleStartTime
         ) {
             return Phase.Allowlist;
+        } else if (
```

Comment:
Could you also return `Ended` when the collection is fully minted ? Just for consistency

Reply:
done above - should we use `totalSupply()` or `_totalMinted()`?

Reply:
`totalSupply` is fine I think